### PR TITLE
Add sync-scroller example

### PR DESCRIPTION
**Pull request by majido**
Add a demo where one scroller offset is synced off of another one (similar to sublime text code editor)

Some minor changes:
- Add scroll offset setter for AnimationProxy
- Fix bug where we were not waiting for proxy to become initialized

### CODE REVIEW

File: animation-worklet/anim-worklet-worker.js [hunk 1]
```diff
@@ -26,14 +26,23 @@ limitations under the License.
...
   };
 
   class ProxyElementWrapper {
     constructor(desc) {
+      this.proxy_ = desc.proxy;
       this.scrollOffsets = new ScrollOffsetsWrapper(desc.proxy);
       this.styleMap = new StyleMapWrapper(desc);
     }
+
+    get ready() {
```
Comment (flackr) [inline animation-worklet/anim-worklet-worker.js]: Thanks for fixing this, I don't see ready() as part of the proposal at https://github.com/majido/animation-worklet-proposal), can we suffix this with "_" to imply that it is private? See https://google.github.io/styleguide/javascriptguide.xml?showone=Naming#Naming

File: animation-worklet/sync-scroller/index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+<script src="../anim-worklet.js"></script>
+<script>
+  var template = document.getElementById('code');
+  document.getElementById('master_scroller').appendChild(template.content.cloneNode(true));
+  document.getElementById('preview_scroller').appendChild(template.content.cloneNode(true));
+
+  setTimeout(function() {
+    window.polyfillAnimationWorklet.import('sync-scroller-animator.js');
+
+  }, 100);
```
Comment (flackr) [inline animation-worklet/sync-scroller/index.html]: Why is this delayed?

**Review by flackr — CHANGES_REQUESTED** [review]
Just a few comments, but looks good overall.

File: animation-worklet/anim-worklet-worker.js (hunk 1 above)
Comment (majido) [reply animation-worklet/anim-worklet-worker.js]: replaced with |ready_|. I also changed the computation of ready status for all values to a functional style.

File: animation-worklet/sync-scroller/index.html (hunk 2 above)
Comment (majido) [reply animation-worklet/sync-scroller/index.html]: It is unnecessary. Removed.

**Comment by majido** [comment]
I also added an additional demo for spring timing.

**Pull request merged**